src/config/constants/tokens.ts

```diff
@@ -51,6 +51,13 @@ export const ETH = new Token(
   'ETH',
   'Binance-Peg Ethereum Token',
 )
+export const USDC = new Token(
+  ChainId.MAINNET,
+  '0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d',
```

Comment:
Should we agree to use checksummed addresses for everything? Token-lists are checksummed, might cause some problems in case `===` is used